content/apps/quotas.md

```diff
@@ -2,64 +2,14 @@
 menu:
   main:
     parent: apps
-title: Quotas
+title: Application Quotas
```

Comment:
Just what it says in the page... These are the specific settings for `memory`, `disk_quota`, and `instances` that you'd typically put in your manifest, or specify via `cf` with flags `-m`, `-k`, or `-i`. I suppose we need yet another page for `space-quota`.